### DISCUSSION TITLE
node/node%E6%96%87%E6%A1%A3/DNS

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#951] [Announcements]
# node/node%E6%96%87%E6%A1%A3/DNS

DNS Node.js 中的 DNS 模块允许你与网络上的域名系统（Domain Name System）进行交互。简单来说，当你想要访问一个网站比如www.example.com时，你的计算机需要知道这个网址对应的 IP 地址是什么，因为在网络世界中，计算机是通过 IP 地址来找到并连接彼此的。DNS 就像是一个电话簿，它帮助你将容易记住的域名（例如...

https://doc.cherrychat.org/node/node%E6%96%87%E6%A1%A3/DNS.html

**Comment by fsen-capricorn** [comment]
发现很多都在瞎说。
比如 resolver.on 这个方法根本是没有的。
还有 dns.Resolver 里面的参数其实只有 timeout 和 tries，但是它自己造出来了 servers 参数
建议用新模型再洗一遍应该会好点
